git.lua

```diff
@@ -594,35 +661,35 @@ local git_parser = parser(
         "submodule",
         "subtree",
         "svn"..parser({
-		    "init"..parser("-T", "--trunk", "-t", "--tags", "-b", "--branches", "-s", "--stdlayout",
```

Comment:
Please remove this change. The whitespace changes is not a part of feature, you adding :)